src/@awesome-cordova-plugins/plugins/purchases/index.ts

```diff
@@ -343,30 +490,98 @@ export class Purchases extends AwesomeCordovaNativePlugin {
     return;
   }
 
+  /**
+   * Make a purchase
+   *
+   * @param {PurchasesStoreProduct} product The product you want to purchase
+   * @param {GoogleProductChangeInfo} googleProductChangeInfo Android only. Optional GoogleProductChangeInfo you
+   * wish to upgrade from containing the oldProductIdentifier and the optional prorationMode.
+   * @param {boolean} googleIsPersonalizedPrice Android and Google only. Optional boolean indicates personalized pricing on products available for purchase in the EU.
+   * For compliance with EU regulations. User will see "This price has been customize for you" in the purchase dialog when true.
```

Comment:
typo customize -> customized

```suggestion
   * For compliance with EU regulations. User will see "This price has been customized for you" in the purchase dialog when true.
```